### PR TITLE
Add fuzzer tests.

### PR DESCRIPTION
**Pull request by floitsch**

### CODE REVIEW

**Comment by floitsch** [comment]
Tests are copied verbatim.
Added copyright.
Added it to the `make`.

File: tests/fuzzer/afl-tests/literals.toit [hunk 1]
```diff
@@ -0,0 +1,10 @@
+main:
```
Comment (kasperl) [inline tests/fuzzer/afl-tests/literals.toit:1]: I assume these files cannot have the copyright header + license?

File: tests/fuzzer/lib/core/core.toit [hunk 2]
```diff
@@ -0,0 +1,77 @@
+// Copyright (C) 2020 Toitware ApS. All rights reserved.
+
```
Comment (kasperl) [inline tests/fuzzer/lib/core/core.toit:2]: License?

**Review by kasperl — APPROVED** [review]
Looks good to me.

File: tests/fuzzer/lib/core/core.toit (hunk 2 above)
Comment (floitsch) [reply tests/fuzzer/lib/core/core.toit:2]: Licenses will be done in a later PR.

File: tests/fuzzer/afl-tests/literals.toit (hunk 1 above)
Comment (floitsch) [reply tests/fuzzer/afl-tests/literals.toit:1]: The fuzzer would use it as input. So theoretically possible.
Will think about it when adding the copyright headers.

**Pull request merged**